scripts/run-draft-pr-scripts.ts

```diff
@@ -131,48 +150,33 @@ class DraftPRScriptRunner {
       }
       
       console.log(`✅ Found draft PR #${pr.number} for branch: ${this.branchName}`);
+      console.log(`✅ Found PR #${pr.number}, using current HEAD for script discovery`);
       return pr.number.toString();
     } catch (error) {
       throw new Error(`Failed to find PR for branch ${this.branchName}: ${error}`);
     }
   }
 
-  private async setupPREnvironment(): Promise<void> {
-    console.log(`🔍 Finding PR for branch: ${this.branchName}...`);
-    
-    this.prNumber = await this.findPRByBranch();
-    
-    console.log(`🔄 Checking out PR #${this.prNumber} branch...`);
-    
-    const githubToken = process.env.GITHUB_TOKEN;
-    const githubRepo = process.env.GITHUB_REPOSITORY;
-    
-    if (!githubToken || !githubRepo) {
-      console.log("⚠️ Not in GitHub Actions environment, skipping PR checkout");
-      return;
-    }
-    
+
+
+  private getAddedFilesInPR(): string[] {
```

Comment:
Ok instead of using `git`, remove all of the git commands and get the added files using GitHub's api:

GET /repos/:owner/:repo/pulls/:pull_number/files